hooks/useGlobalHistoricalDebtData.ts

```diff
@@ -20,18 +21,27 @@ const useGlobalHistoricalDebtData = () => {
 		data: [],
 	});
 
-	const { useDailyStakingActivityQuery } = useSynthetixQueries();
+	const issuanceURL =
+		React.useContext(SynthetixQueryContext)?.context.subgraphEndpoints.issuance || '';
+	const dailyIssued = issuance.useGetDailyIssueds(
+		issuanceURL,
+		{ orderBy: 'id', orderDirection: 'desc' },
+		{ id: true, totalDebt: true }
+	);
+	const dailyBurned = issuance.useGetDailyBurneds(
+		issuanceURL,
+		{ orderBy: 'id', orderDirection: 'desc' },
+		{ id: true, totalDebt: true }
+	);
 
-	const activityQuery = useDailyStakingActivityQuery();
-
-	const isLoaded = activityQuery.isSuccess;
+	const isLoaded = dailyIssued.isSuccess && dailyBurned.isSuccess;
 
 	useEffect(() => {
 		if (isLoaded) {
-			const activity = activityQuery.data ?? [[], []];
+			const activity = [dailyIssued.data ?? [], dailyBurned.data ?? []];
 
 			// We concat both the events and order them (asc)
-			const eventBlocks = orderBy((activity[0] as any).concat(activity[1]), 'timestamp', 'asc');
+			const eventBlocks = orderBy((activity[0] as any).concat(activity[1]), 'id', 'asc');
```

Comment:
Just confirming that this change has the indented behaviour. Is sorting on id the same as sorting on timestamp?

Reply:
yes the old function pulled out the timestamp for you, but in this case we have to use the ID because the codegen doesn't do that.